scss/_variables.scss

```diff
@@ -2282,6 +2282,11 @@ $quantity-selector-btn-padding-x-sm:      add($btn-icon-padding-x-sm, 2px) !defa
 $quantity-selector-icon-width:            .875rem !default;
 $quantity-selector-icon-sm-width:         .625rem !default;
 
+$quantity-selector-disabled-color:        var(--#{$prefix}disabled-color) !default; // Could be `$input-disabled-color` after #2221
```

Comment:
Dark mode PR will prolly be merged before #2221. I'd suggest to add a comment in #2221 to check that there's nothing to modify for these new QS Sass variables and remove this comment here? Thoughts?